README.md

```diff
@@ -170,24 +187,64 @@ Delta computation is based on ModTime of file system and LSN number of pages in
 
 To configure base for next delta backup (only if `WALG_DELTA_MAX_STEPS` is not exceeded). `WALG_DELTA_ORIGIN` can be LATEST (chaining increments), LATEST_FULL (for bases where volatile part is compact and chaining has no meaning - deltas overwrite each other). Defaults to LATEST.
 
-* `WALG_COMPRESSION_METHOD`
+### MySQL
 
-To configure compression method used for backups. Possible options are: `lz4`, 'lzma', 'brotli'. Default method is `lz4`. LZ4 is the fastest method, but compression ratio is bad.
-LZMA is way much slower, however it compresses backups about 6 times better than LZ4. Brotli is a good trade-off between speed and compression ratio which is about 3 times better than LZ4.
+* `WALG_MYSQL_DATASOURCE_NAME`
 
-* `WALG_DISK_RATE_LIMIT`
+To configure connection string for MySQL. Format ```user:password@host/dbname```
 
-To configure disk read rate limit during ```backup-push``` in bytes per second.
+* `WALG_MYSQL_BINLOG_DST`
 
-* `WALG_NETWORK_RATE_LIMIT`
+To place binlogs in the specified directory during stream-fetch.
 
-To configure network upload rate limit during ```backup-push``` in bytes per second.
+* `WALG_MYSQL_BINLOG_SRC`
+
+To configure directory with binlogs for ```binlog-push```.
+
+* `WALG_MYSQL_BINLOG_END_TS`
+
+To set time [RFC3339](https://www.ietf.org/rfc/rfc3339.txt) for recovery point.
+
+* `WALG_MYSQL_SSL_CA`
```

Comment:
Do we need any special options in DSN in order for this SSL config to be used ?

Reply:
@x4m

Reply:
Can you give an example of suggestion options? Because we don't know anything about it.

Reply:
WALG_MYSQL_SSL_CA  implicitly add tls=custom option to DSN.  So you should not specify any  tls option in DSN by yourself